go/service/assets/api_op_CreateAsset_test.go

```diff
@@ -21,26 +20,31 @@ func Test_CreateAsset(t *testing.T) {
 		assert.NoError(t, err, "error while reading request body")
 		assert.JSONEq(t, `
 {
+	"@context": {
+		"edc": "https://w3id.org/edc/v0.0.1/ns/"
+	},
+	"@id": "",
```

Comment:
is there a reason why we need this here?

Reply:
This was coded when we were blocked by an issue cause by asset having different `id` fields ( `@id` and `asset.@id`).  I believe it is fixed now. Need to retest and check.